x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -111,16 +117,37 @@ export const SelectInferenceId = ({
 
   const fieldConfigModelId = getFieldConfig('inference_id');
   const defaultInferenceIds: EuiSelectableOption[] = useMemo(() => {
-    return [{ checked: 'on', label: 'elser_model_2' }, { label: 'e5' }];
+    return [
+      {
+        checked: 'on',
+        label: i18n.translate(
+          'xpack.idxMgmt.mappingsEditor.parameters.inferenceId.popover.defaultModel.Elser',
+          {
+            defaultMessage: 'elser_model_2',
```

Comment:
Do we need to translate 'elser_model_2'? This is the model_id used for default elser model.

Reply:
Correct, good catch -- we shouldn't be translating `id` strings, only things that need to be translated for different languages.